README.md

```diff
@@ -1,26 +1,28 @@
 # Vorlage-Latex
-Dies ist eine LateX Vorlage für Projekt-, Beleg-, Bachelor- und Diplomarbeiten an der Staatlichen Studienakademie Glauchau (BA Glauchau).
 
-Die Vorlage hält sich an die [Hinweise zur Anfertigung wissenschaftlicher Arbeiten (HAWA)](https://www.ba-glauchau.de/fileadmin/glauchau/waehrend-des-studium/dokumente/pruefungen/4BA-F.207_Hinweise_zur_Anfertigung_wissenschaftlicher_Arbeiten.pdf).
 
-Die oben genannten unterstützten Versionen sind jeweils für 1-4 Beteiligte problemlos verwendbar.
+
```

Comment:
```suggestion

















```

Reply:
jk aber mach mal noch die unnötigen newlines weg (nach ner überschrift kann gleich in der nächsten zeile text kommen)
```md
## like
this
```

Reply:
juunge das wird eh nich angezeigt so is die md einfach bisschen übersichtlicher aber okay xD